src/core/utils/StringUtilities.h

```diff
@@ -17,6 +17,7 @@
 #include <stddef.h>
 #include <sstream> // IWYU pragma: keep
 #include <vector>
+#include <boost/optional.hpp>
```

Comment:
is boost optional actually used in this file?

Reply:
Oops, it was temporarily, but not anymore.  Thanks for catching this. Fixed.